### DISCUSSION TITLE
Author List dont show data

### DISCUSSION THREAD

**Discussion by lokize** [hasura/graphql-engine#7424] [General]
Hello friends, I'm new to Hasura and Graphql after following all the steps, creating the data in the database, installing and adding the data from my hasura in the project quasar the data is not listed in my application, I don't know what to do to solve it. the problem!
![image](https://user-images.githubusercontent.com/19701630/130155515-1d8f317b-8b47-42b5-8e12-02d526c596fd.png)

**Comment by motleydev** [comment]
Hi @lokize, sorry to hear about the issue. Are you able to provide some additional details? What did you do to try and connect it? Can you link to the steps you tried following? Thanks!
